velox/exec/RowContainer.h

```diff
@@ -1510,9 +1536,12 @@ class RowContainer {
   // Offset and null indicator offset of non-aggregate fields as a single word.
   // Corresponds pairwise to 'types_'.
   std::vector<RowColumn> rowColumns_;
-  // Optional aggregated column stats(e.g. min/max size) for non-aggregate
+  // Aggregated column stats(e.g. min/max size) for non-aggregate
   // fields. Index aligns with 'rowColumns_'.
   std::vector<RowColumn::Stats> rowColumnsStats_;
+  // Indicates if the min/max column stats are valid. This is set to false
+  // whenever a row is erased.
+  bool rowColumnsStatsMinMaxValid_ = true;
```

Comment:
```
s/minMaxColumnStatsValid_/
```
We need to set it back to true in RowContainer::clear() and please also comment about this